sbin/pfdhcplistener

```diff
@@ -46,7 +46,7 @@ use pf::config::util;
 use pf::services::util;
```

Comment:
This is still not removed from this PR...

Please remove it as it is not relevant anymore